tools/LocTestingSampleApp/LocTestingSampleApp.csproj

```
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
+    <RootNamespace>axe_win_testing</RootNamespace>
```

This doesn't match the code. Maybe
```suggestion
    <RootNamespace>AxeWinLocTesting</RootNamespace>
```